packages/formation-react/src/components/Pagination/Pagination.jsx

```diff
@@ -13,12 +13,16 @@ class Pagination extends React.Component {
     this.pageNumbers = this.pageNumbers.bind(this);
   }
 
+  getAriaLabelSuffix() {
+    return this.props.ariaLabelSuffix || '';
```

Comment:
Would setting a default prop for `ariaLabelSuffix` work? That might be slightly simpler.

Reply:
Should probably bump the package version as well.